### PR TITLE
Passing ViewerNavigation Tests.

### PR DESCRIPTION
**Pull request by phil-plencner-hl**
PR for ViewerNavigation.test.js

### CODE REVIEW

File: __tests__/src/components/ViewerNavigation.test.js [hunk 1]
```diff
@@ -16,125 +15,138 @@ function createWrapper(props) {
...
         viewingDirection: 'bottom-to-top',
       });
-    });
-
-    it('changes the arrow styles', () => {
-      const previous = wrapper.find(MiradorMenuButton).first().children(NavigationIcon).props();
-      const next = wrapper.find(MiradorMenuButton).last().children(NavigationIcon).props();
-      expect(previous.style).toEqual({ transform: 'rotate(90deg)' });
-      expect(next.style).toEqual({ transform: 'rotate(270deg)' });
+      expect(screen.getByLabelText('previousCanvas', { selector: 'button' }).querySelector('svg')).toHaveStyle('transform: rotate(90deg);'); // eslint-disable-line testing-library/no-node-access
```
Comment (cbeer) [inline __tests__/src/components/ViewerNavigation.test.js:148]: I think checking the role is probably preferable to checking by text

```suggestion
      expect(screen.getByRole('button', { name: 'previousCanvas' }).querySelector('svg')).toHaveStyle('transform: rotate(90deg);'); // eslint-disable-line testing-library/no-node-access
```

File: src/components/ViewerNavigation.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ export class ViewerNavigation extends Component {
 
     return (
       <div
+        role="navigation"
```
Comment (cbeer) [inline src/components/ViewerNavigation.js:44]: I'll have to go back and look, but I dimly recall we intentionally decided that a navigation role was inappropriate here.

Comment (phil-plencner-hl) [reply src/components/ViewerNavigation.js:44]: Ok. I'm not sure how else to select that item. The first test might not be necessary (just checking for existence of the wrapper `<div>`). However, there's another test that is specifically checking the viewing direction....
https://github.com/ProjectMirador/mirador/pull/3685/files#diff-69ebf93942b44352912effb10c94e7ae4687632918a4a61526ea1e37a9e7dbdcR122

File: __tests__/src/components/ViewerNavigation.test.js (hunk 1 above)
Comment (phil-plencner-hl) [reply __tests__/src/components/ViewerNavigation.test.js:148]: @cbeer: I don't see a button role on anything in the debug? Changing the line to what you suggested makes the test fail.

Comment (cbeer) [reply __tests__/src/components/ViewerNavigation.test.js:148]: Unless the button has an explicit role, it should have the implicit role `button`:

Comment (phil-plencner-hl) [reply __tests__/src/components/ViewerNavigation.test.js]: @cbeer I was able to update them all to use the button role.

File: src/components/ViewerNavigation.js (hunk 2 above)
Comment (phil-plencner-hl) [reply src/components/ViewerNavigation.js]: @cbeer I updated the tests to not rely on the navigation role. I removed the navigation role from the component.

https://github.com/ProjectMirador/mirador/pull/3685/commits/967dea90aeb2bfba0d684bb8b3c12e878dde7ebd

**Pull request merged**